microk8s-resources/actions/enable.cilium.sh

```diff
@@ -73,6 +74,7 @@ else
       --set global.cni.customConf=true \
       --set global.containerRuntime.integration="containerd" \
       --set global.containerRuntime.socketPath="$SNAP_COMMON/run/containerd.sock" \
```

Comment:
You should also be able to do `--set global.cni.confPath=$SNAP_DATA` above here, and then the `mkdir`, `cp`, `sed` below can be removed.

Reply:
Thanks for the review. That setting is already there. The commands below is just to move the `cni.yaml` to the right place and not the tmp.

Reply:
My point is that they shouldn't be necessary if we just write the config correctly from Helm the first time :-) Originally when I introduced this, the v1.6 YAMLs didn't include an option to configure this option so this is why the script does the copy/sed tricks below.

Any way, this is an unrelated cleanup, up to you on whether you think it makes sense to change it.

Reply:
Oh i know what you meant now. 😁 Sorry kindda slow. Haha thanks for the pointers.  Will update the pr.